Source/Extensions/Blazorise.TreeView/TreeView.razor.cs

```diff
@@ -334,6 +344,11 @@ public TreeViewSelectionMode SelectionMode
     /// </summary>
     [Parameter] public Func<TNode, Task<IEnumerable<TNode>>> GetChildNodesAsync { get; set; }
 
+    /// <summary>
+    /// Indicates the node's disabled state. Used for preventing selection.
+    /// </summary>
+    [Parameter] public Func<TNode, bool> IsDisabled { get; set; } = node => false;

```

Comment:
I think assuming the user can have some logic that eventually can make this null is a better approach.

So instead of setting a default function and assuming this function will always exist, in my opinion this should be null and we should null check whenever we have to call this. Also I believe that's the approach we generally use on Blazorise. @stsrki

Reply:
I'm a little divided by this. It makes sense both ways. But an extra null check feels safer.

Reply:
Done. Just see if the naming for an intermediate makes sense. 

 ```cs
protected Func<TNode, bool> DetermineHasChildNodes => HasChildNodes ?? ( node => false );
 protected Func<TNode, bool> DetermineIsDisabled => IsDisabled ?? ( node => false );
```

Reply:
Ah we usually use the  **Is** or **Has** as that already conveys that it's some logic that will determine a boolean result but this is a naming clash... and per your previous observation for naming clash, this would be **_HasChildNodes** **_IsDisabled**? O.o

Anyway **Determine** is fine by me.

Reply:
Actually, that is a good point. The `_` is a good alternative.

Reply:
In this case, Determine is better because it determines if HasChildNodes is safe, and if not it fallback to default.